client/src/config/downloadOptions.js

```diff
@@ -33,7 +33,7 @@ export const downloadOptions = {
     }
   },
   PROJECT_MULTIPLEXED_ZIP: {
-    header: 'Download Single-cell Data',
+    header: 'Download Multiplexed Data',
```

Comment:
Yeah this is a change that was to help disambiguate this scenario. Since `computed_files.type` will be removed in the future the `header` attribute will be removed soon.

<img width="661" alt="Screen Shot 2024-02-23 at 10 00 03 AM" src="https://github.com/AlexsLemonade/scpca-portal/assets/1075609/27da0768-ff4f-41ba-8a09-ff9cbfe8fcbc">

Reply:
Oh I see! Thank you for letting me know!